controllers/main.go

```diff
@@ -1760,12 +1760,14 @@ func (controller *MainController) RegisterPost(c web.C, r *http.Request) (string
 		return controller.Register(c, r)
 	}
 
-	err = controller.emailSender.Registration(email, controller.baseURL, remoteIP, token.String())
-	if err != nil {
-		session.AddFlash("Unable to send verification email", "registrationError")
-		log.Errorf("error sending verification email %v", err)
-	} else {
-		session.AddFlash("A verification email has been sent to "+email, "registrationSuccess")
+	if controller.getNetworkName() != "testnet" {
```

Comment:
It's okay to allow sending verification emails even on testnet.

Reply:
Sending of verification emails while on testnet has been re-enabled.